lib/client/Client.js

```diff
@@ -305,11 +305,32 @@ export default class Client {
       }
     }).then(function(messagesSet) {
       if (messagesSet.updated.indexOf(id) < 0) {
-        throw new Error('Failed to move message ' + id + ', reason is ' + messagesSet.notUpdated[id]);
+        throw new Error('Failed to move message ' + id + ', the reason is: ' + messagesSet.notUpdated[id]);
```

Comment:
I've changed the message as kader has asked to me.
Add `|| missing` should be done too the next time!